extensions/blocks/map/edit.js

```diff
@@ -191,6 +192,12 @@ class MapEdit extends Component {
 							checked={ scrollToZoom }
 							onChange={ value => setAttributes( { scrollToZoom: value } ) }
 						/>
+						<ToggleControl
+							label={ __( 'Show Fullscreen Button', 'jetpack' ) }
+							help={ __( 'Allow the user to display the map in fullscreen.', 'jetpack' ) }
```

Comment:
Maybe it should say "your visitors" instead "the user"? It sounds a bit friendlier to me.